### PR TITLE
OSDOCS-8683: Removed the hyperthreading parameter from the vSphere docs

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-8693](https://issues.redhat.com/browse/OSDOCS-8683)

Version(s):
4.15 to 4.11

Link to docs preview:
* [Installing a cluster on vSphere with customizations](https://67569--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations#installation-installer-provisioned-vsphere-config-yaml_installing-vsphere-installer-provisioned-customizations)
* [Installing a cluster on vSphere with network customizations](https://67569--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations#installation-installer-provisioned-vsphere-config-yaml_installing-vsphere-installer-provisioned-network-customizations)
* [Installing a cluster on vSphere with user-provisioned infrastructure-changed](https://67569--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere#installation-vsphere-config-yaml_installing-vsphere)
* [Installing a cluster on vSphere with network customizations-changed](https://67569--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-network-customizations#installation-vsphere-config-yaml_installing-vsphere-network-customizations)
* [Installing a cluster on vSphere in a restricted network](https://67569--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere#installation-installer-provisioned-vsphere-config-yaml_installing-restricted-networks-installer-provisioned-vsphere)
* [Installing a cluster on vSphere in a restricted network with user-provisioned infrastructure - changed](https://67569--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-restricted-networks-vsphere#installation-vsphere-config-yaml_installing-restricted-networks-vsphere)
* [Installation configuration parameters-AWS](https://67569--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installation-config-parameters-aws)
* [Installation configuration parameters for GCP](https://67569--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installation-config-parameters-gcp)
* [Installation configuration parameters for Azure](https://67569--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installation-config-parameters-azure)
* [Installation configuration parameters for IBM Power Virtual Server](https://67569--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_powervs/installation-config-parameters-ibm-power-vs)

QE review:
- [x] QE has approved this change.

Additional information:
* We cannot use the [.small] role because it might collide with the CSS "small" syntax. See [Slack thread](https://app.slack.com/client/E030G10V24F?selected_team_id=E030G10V24F#:~:text=It%27s%20an%20arbitrary%20Asciidoctor%20markup%20that%20IMO%20should%20not%20be%20used%20for%20docs.%20it%27s%20not%20that%20it%27s%20deprecated%2C%20just%20that%20%5Bsmall%5D%20maps%20to%20a%20css%20.small%20class%20which%20may%20or%20may%20not%20be%20included%20in%20the%20next%20portal%20build).

### CODE REVIEW

File: modules/installation-vsphere-config-yaml.adoc [hunk 1]
```diff
@@ -4,10 +4,13 @@
 // * installing/installing_vsphere/installing-vsphere-network-customizations.adoc
 // * installing/installing_vsphere/installing-vsphere.adoc
 
-ifeval::["{context}" == "installing-restricted-networks-vsphere"]
-:restricted:
+ifeval::["{context}" == "installing-vsphere"]
+:vsphere:
+endif::[]
+ifeval::["{context}" == "installing-vsphere-network-customizations"]
+:vpshere-custom:
```
Comment (bscott-rh) [inline modules/installation-vsphere-config-yaml.adoc:11]:
```suggestion
:vsphere-custom:
```
typo

File: modules/installation-vsphere-config-yaml.adoc [hunk 2]
```diff
@@ -26,97 +29,62 @@ apiVersion: v1
...
-ifndef::restricted[]
-ifndef::openshift-origin[]
-fips: false <15>
-endif::openshift-origin[]
-ifndef::openshift-origin[]
-pullSecret: '{"auths": ...}' <16>
-endif::openshift-origin[]
-ifdef::openshift-origin[]
+    diskType: thin <13>
+ifdef::vsphere,vpshere-custom[]
```
Comment (bscott-rh) [inline modules/installation-vsphere-config-yaml.adoc:68]:
```suggestion
ifdef::vsphere,vsphere-custom[]
```

File: modules/installation-vsphere-config-yaml.adoc [hunk 3]
```diff
@@ -26,97 +29,62 @@ apiVersion: v1
...
+fips: false <14>
 pullSecret: '{"auths": ...}' <15>
-endif::openshift-origin[]
-endif::restricted[]
+sshKey: 'ssh-ed25519 AAAA...' <16>
+endif::vsphere,vpshere-custom[]
```
Comment (bscott-rh) [inline modules/installation-vsphere-config-yaml.adoc:72]:
```suggestion
endif::vsphere,vsphere-custom[]
```

File: modules/installation-vsphere-config-yaml.adoc [hunk 4]
```diff
@@ -26,97 +29,62 @@ apiVersion: v1
...
       user: administrator@vsphere.local
-    diskType: thin <14>
-ifndef::restricted[]
-ifndef::openshift-origin[]
-fips: false <15>
-endif::openshift-origin[]
-ifndef::openshift-origin[]
-pullSecret: '{"auths": ...}' <16>
-endif::openshift-origin[]
-ifdef::openshift-origin[]
```
Comment (bscott-rh) [inline modules/installation-vsphere-config-yaml.adoc:74]: Correct me if I'm wrong, but it looks like a bunch of content got removed here. Lines 66-74, as well as most of lines 79-107. why was it removed?

File: modules/installation-vsphere-config-yaml.adoc [hunk 5]
```diff
@@ -125,89 +93,53 @@ base and include the cluster name.
...
-`registry.example.com` or `registry.example.com:5000`. For `<credentials>`,
-specify the base64-encoded user name and password for your mirror registry.
-<17> The public portion of the default SSH key for the `core` user in
-{op-system-first}.
+<14> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
 +
-[NOTE]
+[IMPORTANT]
 ====
-For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, specify an SSH key that your `ssh-agent` process uses.
```
Comment (bscott-rh) [inline modules/installation-vsphere-config-yaml.adoc:207]: Why was this SSH note replaced with the FIPS important admonition?

File: modules/installation-vsphere-config-yaml.adoc [hunk 6]
```diff
@@ -219,26 +151,18 @@ specify the base64-encoded user name and password for your mirror registry.
...
 endif::restricted[]
 
-ifeval::["{context}" == "installing-restricted-networks-vsphere"]
-:!restricted:
+ifeval::["{context}" == "installing-vsphere"]
+:!vsphere:
 endif::[]
-ifdef::openshift-origin[]
+ifeval::["{context}" == "installing-vsphere-network-customizations"]
+:!vpshere-custom:
```
Comment (bscott-rh) [inline modules/installation-vsphere-config-yaml.adoc:164]:
```suggestion
:!vsphere-custom:
```

**Review by bscott-rh — CHANGES_REQUESTED** [review]

File: modules/installation-vsphere-config-yaml.adoc (hunk 4 above)
Comment (dfitzmau) [reply modules/installation-vsphere-config-yaml.adoc:74]: A lot of these lines were repeats that stemmed around the `openshift-origin` variable, which seems defined at a higher level? I removed this variable from the docs and, instead, created more direct variables that target specific modules. I had the luxury of restructuring this file as only 3 vSphere modules are impacted.

File: modules/installation-vsphere-config-yaml.adoc (hunk 5 above)
Comment (dfitzmau) [reply modules/installation-vsphere-config-yaml.adoc:207]: Line numbers changed based on the removed duplicate lines, but the notes still target the same callouts. Please let me know if I misunderstood the step out from the original content.

**Review by bscott-rh — CHANGES_REQUESTED** [review]
Just the one formatting issue, then LGTM. Thank you for restoring the conditionals

**Review by bscott-rh — COMMENTED** [review]
LGTM.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Review by bscott-rh — APPROVED** [review]

**Pull request merged**